clusterloader2/pkg/measurement/common/waitforrcs.go

```diff
@@ -0,0 +1,92 @@
+/*
```

Comment:
Could you rename this file to `wait_for_rc_pods.go` to make it more clearer?
Also rename the above file to `wait_for_pods.go`.

Reply:
This will be done in separate PR. The problem is that if I'm changing this name, I should also rename all other files to keep same convention.

Reply:
Sure. Though I don't think it's needed to rename other files (just these 2) to make the names more clearer. I'm not sure of a convention either (and have seen files named either ways).

Reply:
TBH, I will be very unhappy to have some files with `_` and some without. I think that we should stick to only convention for all the files.

Reply:
Sure, I'm ok with `_` everywhere then.

Reply:
Changing names PR: #207